drivers/node/node.go

```diff
@@ -123,6 +123,9 @@ type Driver interface {
 	// RunCommand runs the given command on the node and returns the output
 	RunCommand(node Node, command string, options ConnectionOpts) (string, error)
 
+	// RunCommandWithoutRetry runs the given command on the node but with no retry
+	RunCommandWithoutRetry(node Node, command string, options ConnectionOpts) (string, error)
```

Comment:
[nit] I might prefer `NoRetry`, cuz while I'm still waking up I misread this as `RunCommandWithRetry` (the exact opposite) for a couple files until I did a double-take :joy:

Half a serious nit to prevent misreadings, but feel totally free to leave it as-is